README.md

```diff
@@ -80,39 +76,28 @@ Apache ShardingSphere includes 3 independent products: JDBC, Proxy & Sidecar (Pl
 [![Docker Pulls](https://img.shields.io/docker/pulls/apache/sharding-proxy.svg)](https://store.docker.com/community/images/apache/sharding-proxy)
 
-ShardingSphere-Proxy is a transparent database proxy, providing a database server that encapsulates database binary protocol to support heterogeneous languages. Friendlier to DBAs, the MySQL version now provided can use any kind of terminal (such as MySQL Command Client, MySQL Workbench, etc.) that is compatible with the MySQL protocol to operate data.
+A transparent database proxy, providing a database server that encapsulates database binary protocol to support heterogeneous languages. Friendlier to DBAs, the MySQL version now provided can use any kind of terminal (such as MySQL Command Client, MySQL Workbench, etc.) that is compatible with the MySQL protocol to operate data.
 It can be used directly as MySQL and PostgreSQL servers, and is applicable to any kind of terminal that is compatible with MySQL and PostgreSQL protocol.
 
-![](./docs/document/content/Read-Me-Pics/Read-me-proxy.png)
-
-
-|                         | *ShardingSphere-JDBC* | *ShardingSphere-Proxy* |
-| ----------------------- | --------------------- | ---------------------- |
-| Database                | Any                   | MySQL/PostgreSQL       |
-| Connections Count Cost  | High                  | Low                    |
-| Supported Languages     | Java Only             | Any                    |
-| Performance             | Low loss              | Relatively High loss   |
-| Decentralization        | Yes                   | No                     |
-| Static Entry            | No                    | Yes                    |
+For more details, follow this [link to the official website](https://shardingsphere.apache.org/document/current/en/overview/).
 
 
 ### Hybrid Architecture
 
-ShardingSphere-JDBC adopts a decentralized architecture, applicable to high-performance light-weight OLTP application developed with Java. ShardingSphere-Proxy provides static entry and all languages support, suitable for OLAP application and sharding databases management and operation.
+ShardingSphere-JDBC adopts a decentralized architecture, applicable to high-performance light-weight OLTP applications developed with Java. ShardingSphere-Proxy provides static entry and all languages support, suitable for OLAP application and sharding databases management and operation.
 
 Through the mixed use of ShardingSphere-JDBC & ShardingSphere-Proxy together with a unified sharding strategy by the same registry center, the ShardingSphere ecosystem can build an application system suitable to all kinds of scenarios.
 
-![](./docs/document/content/Read-Me-Pics/Read-me-hybrid.png)
+More details can be found following this [link to the official website](https://shardingsphere.apache.org/document/current/en/overview/).
 
 ## Roadmap:
```

Comment:
It is suggested to place `Solution` above `Roadmap` to connect closely to the products introduced above.

Reply:
Changed